### DISCUSSION TITLE
Timeline for stable release

### DISCUSSION THREAD

**Discussion by harblaith7** [nuxt/nuxt#17918] [General]
Do you guys have an estimated date range for when the stable release of Nuxt 3 will be out?

**Comment by batmanhit** [comment]
I have the same question as @harblaith7 on this topic.
When is the estimated stable release?

We are planning to setup a new project that we are going to develop in the next 6 months and we are looking forward starting with nuxt3.

But not sure when it could be stable and `hybrid` feature is available.

**Comment by pi0** [comment]
We are targetting to start RC releases of Nuxt3 until January. hybrid feature (RFC: https://github.com/nuxt/framework/discussions/560) is waiting for Nitropack project (RFC: https://github.com/nuxt/framework/discussions/1690). Expect it by end of the year :)

**Comment by ghonchesefidi** [comment]
@pi0 Thanks for your reply. Our company is waiting to start a new project with Nuxt3. Do you think, should we still hold the project or start it with Nuxt 2? What is your advice?

**Reply by ghost** [reply]
Hi, I also have the same issue. even Nuxt 3 stable releases tomorrow, it takes some months for the whole ecosystem to adopt it. the modules have to be compatible with it, the Nuxt documentation takes some time to explain everything, the Nuxt itself also takes time to mature, fix the bugs and add mentioned features, etc. even with all of that the community is very important. you may be stuck sometimes and it would be easier if plenty of tutorials, articles, Q&As be available. so if you want to start something big I suggest going with Nuxt 2.
this is just my opinion.

**Reply by ghonchesefidi** [reply]
Thanks for your share. I think the best way to start a new project is using composition API with nuxt 2 so we don't need to change lots of things on migrating to Nuxt3. What do you think about this strategy?

**Reply by TheAlexLichter** [reply]
Starting with Nuxt2 and migrating later will likely work better (e.g. because you can already gather feedback regarding the product itself in the meantime).

**Reply by ghost** [reply]
I agree with @manniL.

**Reply by zhaoweizhong** [reply]
But there are some packages I want to use in my project, and they only support Vue 3...

**Reply by TheAlexLichter** [reply]
Yeah, I feel you. Especially [HeadlessUI](https://headlessui.dev/) 😆. 
Then you only have three options:
* Start with plain Vue3 and migrate to Nuxt at one point (but migration won't be as easy as Nuxt2 -> Nuxt3)
* Try to build it with Nuxt3 (but some modules are still missing and it is in beta as you know)
* Do not use the libs/packages yet and use them after migrating from Nuxt2 -> Nuxt3

**Reply by eMeRiKa13** [reply]
One more feedback: I’m building a saas platform, using nuxt3 directly, because I wanted to use vue3 to be future proof. 

Even if I’m a beginner on vue and nuxt, I still enjoy my choice. Routing, server side on nuxt3 are amazing. There are few ready modules, ressources and tutorials but it slowly grows (check the nuxt github discussions for example). In two or three months, I think it will be far better to have a nuxt3 project in development rather than a nuxt and vue2 one!

(Not sure of your point about Headless UI, if it’s nuxt3 ready or not. I’m using it in my nuxt3 project without problems yet.)

**Comment by vencho-mdp** [comment]
Any updates on this?

**Reply by reinoldus** [reply]
![image](https://user-images.githubusercontent.com/2091290/149084782-aa936014-5cff-43ee-8d58-5eb052e4724f.png)

https://twitter.com/nuxt_js/status/1478740961803096068

This is all I know

**Reply by iingles** [reply]
I want to start building my app by March 1st, hoping it's ready by then.  Might just start building with the beta in the meantime.

**Comment by TheAlexLichter** [comment]
The first release candidate is planned for March 2022 ([source](https://twitter.com/Atinux/status/1492134698629451782))

**Comment by zhaoweizhong** [comment]
Will there be a command-line tool for creating a Nuxt 3 app, just like https://github.com/nuxt/create-nuxt-app for Nuxt 2?

**Comment by vencho-mdp** [comment]
Nuxi has some [templates ](https://github.com/nuxt/starter) but not a CLI yet AFAIK.
